_typos.toml

```diff
@@ -1,2 +1,3 @@
 [default.extend-words]
 acn = "ACN"
+datas = "datas"
```

Comment:
"datas" does indeed not exist, but they seem to have used it as an argument name, so even renaming our own variable to "data" would probably not help (though you may try)

Reply:
Correct. I tried to take the typos fix as-is, but it did not work. Since it is an argument name, there is not much we can do. That's why I added it to this file.
I have not used `typos` before. I tried to find out if it is possible to configure it on a per-file or per-line basis, but didn't see a way.

Reply:
That's ok, I just wasn't sure if it checks existing argument names.